### DISCUSSION TITLE
No funciona

### DISCUSSION THREAD

**Discussion by Pakoko1** [tomer8007/whatsapp-web-incognito#93] [Q&A]
Desde la ultima actualizacion de WhatsApp  no funciona, no se ven ni el En linea de los contactos , ni la hora de la ultima conexion , y la verdad creo que no soy el unico que le pasa.

**Comment by tomer8007** [comment]
You can't see the online and last seen of people while you're blocking yours. This is a known limitation. Please see the README.

**Reply by Pakoko1** [reply]
Yo no tengo bloqueado a nadie, está habilitado las notificaciones  de lectura y absolutamente todo, convendria que revisarais la extensjon por que no funciona.... y repito si no lo sabeis comprobarlo
